stl/inc/xstring

```diff
@@ -2179,7 +2179,7 @@ public:
     using reference       = value_type&;
     using const_reference = const value_type&;
 
-    _String_val() : _Bx(), _Mysize(0), _Myres(0) {}
+    _String_val() noexcept : _Bx(), _Mysize(0), _Myres(0) {}

```

Comment:
This change is notable because `_Container_base` allocates a proxy object in debug mode, so exceptions will slam into `noexcept` here. However, no change requested - `basic_string`'s default ctor has already been marked `noexcept` (like `vector`), so there is no real behavioral difference.